.pipelines/templates/release-MakeBlobPublic.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           Write-Host 'AzureRM module removed'
         }
```

Comment:
So all of the above logic here, can we verify if it's necessary (and if so, document why)?